west.yml

```diff
@@ -37,7 +37,8 @@ manifest:
       groups:
         - hal
     - name: edtt
-      revision: 7dd56fc100d79cc45c33d43e7401d1803e26f6e7
+      url: https://github.com/EDTTool/EDTT.git
+      revision: pull/34/head
```

Comment:
I'm curious to this change: What repo is this referring to, and why does it use a PR as revision when it previously, like all the others, used a commit hash instead?

Reply:
That is a very provisional change to test here with that other repo PR.
(A change like this should never be merged here. Therefore the DNM label)

That repo, is the original EDTT repo. There is a couple of repos which are replicated in the Zephyr organization (Zephyr follows a policy of replicating any repo which it uses, to ensure they are there in case the original goes offline)